movement/roi/conditions.py

```diff
@@ -0,0 +1,105 @@
+"""Functions for computing condition arrays involving RoIs."""
+
+from collections import defaultdict
+from collections.abc import Sequence
+
+import numpy as np
+import xarray as xr
+
+from movement.roi.base import BaseRegionOfInterest
+
+
+def compute_region_occupancy(
+    data,
+    regions: Sequence[BaseRegionOfInterest],
+) -> xr.DataArray:
+    """Return a condition array indicating if points were inside regions.
+
+    The function returns a boolean DataArray where each element indicates
+    whether a point in the input ``data`` lies within the corresponding RoIs
+    in ``regions``. The original dimensions of ``data`` are preserved, except
+    for the ``space`` dimension which is replaced by the ``occupancy``
+    dimension. The ``occupancy`` dimension has a number of elements equal to
+    the number of RoIs in the ``regions`` argument and it's coordinate names
+    correspond to the names of the given RoIs.
+
+    Parameters
+    ----------
+    data : xarray.DataArray
+        Spatial data to check for inclusion within the ``regions``. Must be
+        compatible with the ``data`` argument to :func:`contains_point\
+        <movement.roi.base.BaseRegionOfInterest.contains_point`.
```

Comment:
The closing bracket was missing, leading to incorrect rendering of the link.
```suggestion
        <movement.roi.base.BaseRegionOfInterest.contains_point>`.
```